management-server/cmd/adapter/main.go

```diff
@@ -0,0 +1,29 @@
+/*
+ *  Copyright (c) 2022, WSO2 LLC. (http://www.wso2.org) All Rights Reserved.
+ *
+ *  Licensed under the Apache License, Version 2.0 (the "License");
+ *  you may not use this file except in compliance with the License.
+ *  You may obtain a copy of the License at
+ *
+ *  http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *  Unless required by applicable law or agreed to in writing, software
+ *  distributed under the License is distributed on an "AS IS" BASIS,
+ *  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *  See the License for the specific language governing permissions and
+ *  limitations under the License.
+ *
+ */
+
+package main
+
+import (
+	logger "github.com/sirupsen/logrus"
```

Comment:
I think you can use the logging package as this https://github.com/wso2/apk/pull/95/files#diff-da108357a72497b57c499fecbb0dc410460bd03134d02f0b1b91e8d4fe545453

It's using logging package from "github.com/wso2/apk/adapter/pkg/logging"

Reply:
Updated it to use logger